configs/recognition/tsn/tsn_r50_320p_1x1x8_150e_activitynet_video_flow.py

```diff
@@ -67,7 +65,6 @@
         ann_file=ann_file_train,
         data_prefix=data_root,
         filename_tmpl='flow_{}_{:05d}.jpg',
-        with_offset=True,
```

Comment:
What's the effect of removing this line?

Reply:
When training with `video` file lists, it should be set as False (or it will not run successfully).